client/src/components/categorical/value.js

```diff
@@ -17,13 +17,16 @@ import * as globals from "../../globals";
 import styles from "./categorical.css";
 import { Tooltip } from "@blueprintjs/core";
 
+import { AnnotationsHelpers as AH } from "../../util/stateManager";
```

Comment:
I prefer the longer name here rather than AH, perhaps `AnnoUtil`?

Reply:
This is the style I use elsewhere in the app.  I would rather maintain consistency.  Personally, I think the function name contains all the information, and the package name is not super useful (hence the abbreviated form).   However, if you prefer the verbose form, I can make the mod everywhere in the app.   LMK please.

Reply:
Took a look... I prefer full length for readability, even if verbose. `AnnotationsHelpers` is ok with me. We don't use it in a ton of places.

Reply:
roger!